### ISSUE TITLE
Sock-js net::ERR_NAME_NOT_RESOLVED after upgrading to 9.2.1

### ISSUE THREAD

**Issue by abelovic** [egoist/poi#221]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Getting the following error in the console after upgrading to 9.2.1 (didn't see this before):

```GET http://sockjs-node/info?t=1502212232208 net::ERR_NAME_NOT_RESOLVED```
```[WDS] Disconnected!```

**If the current behavior is a bug, please provide the steps to reproduce.**

My poi.config.js

```
module.exports = (options, req) => ({
  host: '0.0.0.0',
  port: 4000,
  entry: './src/index.js',
  html: {
    template: './src/index.ejs'
  },
  devServer: {
    host: '0.0.0.0',
    port: 4000
  },

....
}
```

**What is the expected behavior?**

Shouldn't get socket errors. Doing a google search it appears as if this happens when the devServer is listening on a different port? 

**Please mention other relevant information such as the browser version, Node.js version, Poi version and Operating System.**

Node 6.11.1
Chrome 60.0
Poi 9.2.1
Windows 10

**Comment by abelovic** [comment]
FYI: I just rolled back all the way to 9.1.7 to see if I could determine when it started. Still see the issue in 9.1.7 so it's either something wrong with my config (I have not changed anything recently though and my environment is the same), or something changed in poi even earlier and didn't realize till now :/

Any ideas on where to start looking?

**Comment by egoist** [comment]
The only change is here https://github.com/egoist/poi/commit/65e667691fecab7e6acf7d9e8d7e13b24d7eac37 can you provide a repo to reproduce this?

**Comment by egoist** [comment]
Reverted that to fix this. try 9.2.2

**Comment by abelovic** [comment]
@egoist - so my project was started along time ago using create-vue-app when this was still vbuild! So I basically started a brand new project from scratch and copied my existing files in. No more issues. Very weird because they were the same files (including config and all dependencies). Can't explain it but something must have been leftover from upgrading :( I was also getting javascript out of memory errors every now and then too.

Thanks for your help!

P.S. It looks like you are not promoting create-vue-app anymore? What is the preferred way to get started? I looked through all the docs and couldn't find anything so I did it by scratch. I also looked at just using a vue-cli template like webpack but is there much value to this since it has its own build mechanism baked in?

I think the create-vue-app is useful but it is still creating vbuild (branded) config (and not updated in months). Didn't really test it to see if it works properly though. I knew how to put the files together by scratch but new people to the project wouldn't so this might be helpful to keep updated (if its not on your list already)
